src/interaction/selection.ts

```diff
@@ -18,15 +18,24 @@ export function start(): void {
 function recompute(): void {
     let s = document.getSelection();
 
-    if (selection !== null && data.start !== null && data.start !== getId(s.anchorNode)) {
+    // Bail out if we don't have a valid selection
```

Comment:
Is this what happens when a user clears their selection? Should we send an update to indicate that?

Reply:
Since selection is not sent as part of an event property, like other events, and we have to poll for the selection after the fact - there's always a chance that we are out of date. We are just ensuring that we are not dealing with a stale selection here.